src/backend/dx12/src/device.rs

```diff
@@ -1497,6 +1515,30 @@ impl d::Device<B> for Device {
                 };
                 let format = attrib.element.format;
 
+                // Check if we need to add a new remapping in-case the offset is
+                // higher than the vertex stride.
+                // In this case we rebase the attribute to zero offset.
+                let binding = attrib.binding as usize;
+                let stride = vertex_strides[binding];
+                let offset = attrib.element.offset;
+                let (input_slot, offset) = if stride <= offset {
```

Comment:
how does it handle a case when `offset < stride` but `offset + element_size > stride`?

Reply:
Works fine so far, added warden test for this.